.tipi/opts.vs-16-2019-win64-cxx17

```diff
@@ -0,0 +1 @@
+add_compile_options(-bigobj)
```

Comment:
What happens when the user wants to build with C++11?

Reply:
Hi @bloomen

we're generally leaning torwards C++17 and later, so by default we currently don't ship a C++11 toolchain with tipi. If however someone wanted to compile for such a target they would have to provide an environment/toolchain-file and add 

```cmake
add_compile_definitions(TRANSWARP_CPP11)
```

...in their own project